riscv_isac/cgf_normalize.py

```diff
@@ -429,21 +429,32 @@ def expand_cgf(cgf_files, xlen):
     '''
 
     cgf = utils.load_cgf(cgf_files)
+    if list_duplicate:
+        print("Found duplicate coverpoints: ")
     for labels, cats in cgf.items():
         if labels != 'datasets':
+            if list_duplicate:
+                print("{0} :".format(labels))
+            count = 0
             for label,node in cats.items():
                 if isinstance(node,dict):
                     if 'abstract_comb' in node:
                         temp = node['abstract_comb']
                         del node['abstract_comb']
                         for coverpoints, coverage in temp.items():
-                            i = 0
                             try:
                                 exp_cp = eval(coverpoints)
                             except Exception as e:
                                 pass
                             else:
                                 for cp,comment in exp_cp:
+                                    prev_len = len(cgf[labels][label])
                                     cgf[labels][label].insert(1,cp,coverage,comment=comment)
+                                    if prev_len == len(cgf[labels][label]):
```

Comment:
The check for redundant coverpoints should be performed only if the feature is enabled. It would probably be usefull to implement it as a utility function which identifies all the redundant coverpoints for a particular function in one go.

Reply:
A CLI option will also be necessary in ISAC to control this feature.